src/components/home/homeMidSection/homeMidSection.tsx

```diff
@@ -1,114 +1,242 @@
-import { Row, Col, Carousel, Button } from "react-bootstrap";
-import projects from "./communities&standards.json";
-import bundeslaender from "../../../assets/homepage/Bundeslaender.svg";
-import 'react-perfect-scrollbar/dist/css/styles.css';
-import PerfectScrollbar from 'react-perfect-scrollbar'
+import { faCircle, faUser } from "@fortawesome/free-regular-svg-icons";
+import {
+  faChartColumn,
+  faDna,
+} from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import React from "react";
+import { Card, Col, Row } from "react-bootstrap";
+import { getMetadataSummary, querySearchService } from "../../../api/browse";
+import { metadataSummaryModel, searchResponseModel } from "../../../models/dataset";
+import { getItemsForSummary } from "../../../utils/utils";
 
 const HomeMidSection = () => {
+  const [searchResults, setSearchResults] =
+    React.useState<searchResponseModel | null>(null);
+  const [summary, setSummary] = React.useState<metadataSummaryModel | null>(
+    null
+  );
+
+  React.useEffect(() => {
+    const getData = () => {
+      querySearchService(setSearchResults, [], "", 0, 1, "Dataset");
+      getMetadataSummary(setSummary);
+    };
+    getData();
+  }, []);
```

Comment:
Use the summary call to get Dataset count

Reply:
Done